ui/components/app/collectible-details/collectible-details.js

```diff
@@ -173,7 +173,11 @@ export default function CollectibleDetails({ collectible }) {
             className="collectible-details__card"
           >
             {image ? (
-              <img className="collectible-details__image" src={image} />
+              <img
+                className="collectible-details__image"
+                src={image}
+                alt={description}
```

Comment:
@darkwing - should we have a fallback `alt` when `description` is unavailable? Perhaps we can do something with the `tokenId`

Reply:
Agreed with @NidhiKJha !

Reply:
Thanks @darkwing and @NidhiKJha! I've updated this PR with fallback text